src/jit/utils.cpp

```diff
@@ -277,20 +277,20 @@ const char* getRegNameFloat(regNumber reg, var_types type)
 #define REGDEF(name, rnum, mask, sname) "x" sname,
 #include "register.h"
     };
-#ifdef FEATURE_AVX_SUPPORT
+#ifdef FEATURE_SIMD
```

Comment:
should the `regNamesYMM` be under `#ifdef` at all?

Reply:
Probably not, though for now they are only used when `FEATURE_SIMD` is present (because they only apply to 256-bit operands which we won't otherwise have). However, when we add intrinsics support, they probably shouldn't be under `FEATURE_SIMD` since the 256-bit intrinsics support will presumably be orthogonal to `FEATURE_SIMD`